publicHtml/ajax.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+const express = require('express');
+const app = express();
+
+app.listen(3619);
+app.use(express.static('public'));
+
+app.get('/XMLHttpRequest', (req, res) => {
+  fs.appendFile('/home/pdiddy/garageScript/publicHtml/public/xmlpublicreq.txt',`\n Name: ${req.query.name} Comment: ${req.query.comment} \n`);
```

Comment:
Optional: Make a constant for the path, message. It will make life easier later.